docs: Try to make the welcome page more exciting (#3769)
ignite/cli: docs/docs/01-welcome/01-index.md

- **Comprehensive Scaffolding:** Easily scaffold modules, messages, CRUD operations, IBC packets, and more, expediting the development of complex functionalities.
- **Development with Live Reloading:** Start and test your blockchain node with real-time updates, enhancing your development workflow.
- **Frontend Flexibility:** Utilize pre-built templates for Vue.js, React, Typescript or Go, catering to diverse frontend development needs.
- **Inter-Blockchain Communication (IBC):** Seamlessly connect and interact with other blockchains using an integrated IBC relayer, a key feature of the Cosmos SDK.
- **CometBFT Integration:** Built with the recently rebranded CometBFT (formerly Tendermint), ensuring robust consensus mechanisms in your blockchain solutions.
- **Cross-Domain Applications:** Ideal for developing a wide range of blockchain applications in sectors like DeFi, NFTs, supply chain management, smart contracts (EVM or WASM), and decentralized exchanges (DEXes).

## Install Ignite CLI

Get started with Ignite CLI by running this simple installation command:

```
curl https://get.ignite.com/cli! | bash
```

## Embracing the Cosmos Ecosystem

> Ignite CLI is your entry point into the vibrant Cosmos ecosystem, a hub of innovation where you can explore a plethora of applications, from wallets and explorers to smart contracts and DEXes, all powered by CometBFT and the Cosmos SDK.

Reviewer: ```suggestion
Ignite CLI is your entry point into the vibrant Cosmos ecosystem, a hub of innovation where you can explore a plethora of applications, from wallets and explorers to smart contracts and DEXes, all powered by CometBFT and the Cosmos SDK.
```

> Ignite CLI is your entry point into the vibrant Cosmos ecosystem, a hub of innovation where you can explore a plethora of applications, from wallets and explorers to smart contracts and DEXes, all powered by CometBFT and the Cosmos SDK.

Reviewer: ```suggestion
Ignite CLI is your entry point into the vibrant Cosmos ecosystem, a hub of innovation where you can explore a range of applications, from wallets and explorers to smart contracts and DEXes, all powered by CometBFT and the Cosmos SDK.
```

Reviewer: or something simpler than "plethora"

Reviewer: May we also add links to CometBFT and Cosmos SDK

> Ignite CLI is your entry point into the vibrant Cosmos ecosystem, a hub of innovation where you can explore a plethora of applications, from wallets and explorers to smart contracts and DEXes, all powered by CometBFT and the Cosmos SDK.

Reviewer: ```suggestion
Ignite is your entry point into the vibrant [Cosmos](https://cosmos.network/) ecosystem, a hub of innovation where you can explore a plethora of applications, from wallets and explorers to smart contracts and DEXes, all powered by [CometBFT](https://cometbft.com/) and the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk).
```

> This ecosystem is home to over [$50 billion worth of blockchain projects](https://cosmos.network/ecosystem/tokens/), showcasing the scalability and versatility of the technologies at play.

Reviewer: ```suggestion
Ignite is a cornerstone in the Cosmos ecosystem, fostering the development of some of the world's most valuable chains.
```

Author: I like the former more. It's convincing that the Cosmos ecosystem secures more than 50B of funds.

> This ecosystem is home to over [$50 billion worth of blockchain projects](https://cosmos.network/ecosystem/tokens/), showcasing the scalability and versatility of the technologies at play.

Reviewer: I'd just state "some of the world's most valuable chains" and not mention valuations.

Reviewer: The latter part, "showcasing the scalability and versatility of the technologies at play", is nice but should be more direct to reflect Ignite.

Author: I thought that's actually the "banger" here. I doubt it will drop but higher valuation is not a problem here
